src/OrderFormAcceptanceTests.Actions/Pages/OrderForm.cs

```diff
@@ -610,6 +653,15 @@ public void EnterProposedDate(string year, string month, string day)
 			Driver.EnterTextViaJs(Wait, Pages.OrderForm.OrderDateYear, year);
 		}
 
+		public string GetProposedDate()
+		{
+			Wait.Until(d => d.FindElements(Pages.OrderForm.OrderDateDay).Count == 1);
+			var day = Driver.FindElement(Pages.OrderForm.OrderDateDay).Text;
+			var month = Driver.FindElement(Pages.OrderForm.OrderDateMonth).Text;
+			var year = Driver.FindElement(Pages.OrderForm.OrderDateYear).Text;
+			return string.Join(" ", day, month, year);
```

Comment:
Why are we returning the date as a string rather than a datetime?

Reply:
make the assertion easier in the steps, and to avoid the time component which may be set from the DB we'd need to convert to string anyway

Reply:
righto, perfectly reasonable